### PR TITLE
Gdr 2050

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Optimized unit tests by removing duplicated tests that have been already tested within https://github.com/gdrplatform/gDRcore/blob/master/tests/testthat/test-convert_se_to_raw_data.R (small and 2 dose nonoise) and by reducing number of drugs and cell lines in gDR pipeline
Related JIRA issue: 

## Why was it changed?

# Checklist for sustainable code base
- [X] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by gladkia — APPROVED** [review]

File: NAMESPACE [hunk 1]
```diff
@@ -45,4 +45,4 @@ import(checkmate)
 importFrom(data.table,".I")
```
Comment (j-smola) [inline NAMESPACE:45]: side note: do we have to import these additionally as we import `data.table` (see: Imports in DESCRIPTION)

File: NEWS.md [hunk 2]
```diff
@@ -1,536 +1,441 @@
...
-#### [0.1.20] - 2020-08-26
-
-##### Added
+# Change to v.0.1.20 - 2020-08-26
 - RDS files for the other GDS data (GDS5, GDS6, GDS7)
 
-#### [0.1.19] - 2020-08-03
-
-##### Updated
+# Change to v.0.1.19 - 2020-08-03
```
Comment (j-smola) [inline NEWS.md:422]: Shouldn't be that somehow only one level list?

**Review by DominikZabinski — APPROVED** [review]

File: tests/testthat/test-normalize_SE.R [hunk 3]
```diff
@@ -50,4 +50,17 @@ test_that("normalize_SE works as expected", {
   se2 <- normalize_SE(se, "single-agent", nested_confounders = 
                         c(gDRutils::get_SE_identifiers(se, "barcode", simplify = TRUE), "masked"))
   expect_s4_class(se2, "SummarizedExperiment")
+  
+
```
Comment (j-smola) [inline tests/testthat/test-normalize_SE.R:54]: I miss some description what exactly we are testing here (but it may be for another task)

**Review by j-smola — APPROVED** [review]
Please, check all examples. Does this line works? `mae <- gDRutils::get_synthetic_data("finalMAE_small")` Probably it is a problem with required versions of gDRutils & gDRtestData

**Pull request merged**